src/config/db.config.js

```diff
@@ -0,0 +1,9 @@
+// change the info below to access to your localhost
```

Comment:
We may want to change this to dummy data for user and password

```
USER: "[YOUR_USERNAME]",
PASSWORD: "[YOUR_PASSWORD]"
```

Something like that

Reply:
fixed